mne/io/edf/edf.py

```diff
@@ -550,12 +559,6 @@ def _get_edf_info(fname, stim_channel, annot, annotmap, tal_channel,
             info['ch_names'][idx] = chan_info['ch_name']
             if isinstance(stim_channel, str):
                 stim_channel = idx
-        if tal_channel == idx:
-            chan_info['range'] = 1
-            chan_info['cal'] = 1
-            chan_info['coil_type'] = FIFF.FIFFV_COIL_NONE
-            chan_info['unit'] = FIFF.FIFF_UNIT_NONE
-            chan_info['kind'] = FIFF.FIFFV_MISC_CH
```

Comment:
sorry, this is temporary until I debug how to fix the offset when it's applied to the tal_channel. as it is, the tal_channel crashes because its values are changed and it affects the decoding.